src/Engine/ProtoCore/Reflection/Mirror.cs

```diff
@@ -66,7 +66,7 @@ public RuntimeMirror(string varname, int blockDecl, ProtoCore.RuntimeCore runtim
 
                 variableName = varname;
                 blockDeclaration = blockDecl;
-                StackValue svData = deprecateThisMirror.GetValue(variableName, blockDeclaration).DsasmValue;
+                StackValue svData = deprecateThisMirror.GetSymbolValue(variableName);
```

Comment:
How is going to handle variables declared in nested blocks, especially using Imperative block?